controllers/replica_set_controller.go

```diff
@@ -557,6 +557,12 @@ func buildAutomationConfig(mdb mdbv1.MongoDBCommunity, auth automationconfig.Aut
 		SetDataDir(mdb.GetMongodConfiguration().GetDBDataDir()).
 		AddModifications(getMongodConfigModification(mdb)).
 		AddModifications(modifications...).
+		AddProcessModification(func(_ int, p *automationconfig.Process) {
+			p.SetLogRotate(mdb.Spec.AgentConfiguration.LogRotate)
+			if mdb.Spec.AgentConfiguration.SystemLog != nil {
```

Comment:
Since logrotate doesn't work without syslog, shouldn't both setters be under the if?

Reply:
since I only plan to inform the user, I don't think a test is needed here, since I would need to check the logs